frontend/src/components/Result.js

```diff
@@ -0,0 +1,131 @@
+import React, { Component } from "react";
+import App, { users, userdata, user } from "../App";
+import { withRouter } from "react-router";
+import IGlogo from "./img/instagram.png";
+import FBlogo from "./img/fb.png";
+import Twitterlogo from "./img/twitter.png";
+import LIlogo from "./img/linkedin.png";
+import IGPost from "./IGPost";
+
+import {
+  UserSearchServiceClient,
+  UserIdRequest
+} from "../protofiles/usersearch_grpc_web_pb";
+
+export class Result extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      user: this.props.location.state && this.props.location.state.user,
+      posts: []
+    };
+  }
+
+  componentDidMount() {
+    const user = this.state.user;
+    if (!user) {
+      console.log(":(");
+      return;
+    }
+    const client = new UserSearchServiceClient("http://localhost:4000");
+    const userIdRequest = new UserIdRequest();
+    userIdRequest.setUserId(user.id);
+    client.getInstaPostsWithUserId(userIdRequest, {}, (err, response) => {
+      if (err) {
+        console.log(err);
+        return;
+      }
+      console.log(response);
```

Comment:
More descriptive please

Reply:
Deleted the `console.log`, but a short description can be used to describe the log.